conans/client/command.py

```diff
@@ -166,6 +166,34 @@ def test(self, *args):
                               options=options,
                               settings=settings,
                               build_mode=args.build)
+        # Check old tests, format
```

Comment:
maybe put all this code in a function? This function could be define inside the "test" method, something named "warn_if_old_layout", just for isolate a little this checks.